lib/docker.ts

```diff
@@ -14,6 +14,8 @@ interface DockerOptions {
   tlsKey?: string;
 }
 
+const SystemDirectories = ["dev", "proc", "sys"];
```

Comment:
When running findGlobs on the root of a docker image we want to exclude these system directories as they will definitely not contain any manifest files for container applications.